libnixf/src/Sema/VariableLookup.cpp

```diff
@@ -74,6 +85,23 @@ void VariableLookupAnalysis::emitEnvLivenessWarning(
   }
 }
 
+static constexpr std::array<std::string_view, 2> KnownIdioms = {
+    "pkgs",
```

Comment:
To make `pkgs` work, we'll need to assume `pkgs` == `import nixpkgs {}` below and specially handle it.